layouts/partials/entry/meta/tags.html

```diff
@@ -0,0 +1,11 @@
+{{- if .Params.tags }}
```

Comment:
>... Tags section in the summary isn't starting in a new line (see screenshots).

That's because of span. Use div (block) container for both partials (categories and tags).